config/setupcfg.py

```diff
@@ -15,7 +15,9 @@
 from hpedockerplugin import configuration as conf
 from hpedockerplugin.hpe import hpe3par_opts as plugin_opts
 from oslo_log import log as logging
+import logging as log1
```

Comment:
This can be used as import logging.

Reply:
There are 2 imports one for oslo_log and other for logging which is imported as log1